www/docs/main/quickstart.mdx

```diff
@@ -453,7 +453,7 @@ Let's now move to the client-side code and embrace the power of end-to-end types
 // @filename: client.ts
 // ---cut---
 import { createTRPCProxyClient, httpBatchLink } from '@trpc/client';
-import type { AppRouter } from './server';
+import type { AppRouter } from '../server';
```

Comment:
I think you'll need to update the shiki include path as well, since it thinks they are next to eachother: https://github.com/trpc/trpc/pull/5149/files#diff-11a3230bbcbee64c6622cbcbb2c8bed895b57a8c2b201e39fbcd6b5f156e1737R451-R453

Reply:
Hey @juliusmarminge ,

I have added comment to maintain consistency in markdown file in new commit. Is this what you meant?

According to my understanding, the shiki include path is correct. This more like a "TYPO". 

![Screenshot from 2023-12-15 20-21-01](https://github.com/trpc/trpc/assets/110909159/baccee7e-8917-4895-b7af-32790d13f3a8)

Here you can see that in "Try it out for youself" section the file path is correct.

Let me know, if I misunderstood.

Thanks.